src/tools/dashboard/src/reference.rs

```diff
@@ -151,72 +155,104 @@ fn extract(par_from: &Path, par_to: &Path) -> Vec<(Example, PathBuf)> {
     pairs
 }
 
-/// Prepends the text in `path` with the given `text`.
-fn prepend_text(path: &Path, text: &str) {
-    let code = fs::read_to_string(&path).unwrap();
-    let code = format!("{}\n{}", text, code);
-    fs::write(&path, code).unwrap();
+/// Returns a set of paths to the config files for examples in the Rust books.
+fn get_config_paths() -> HashSet<PathBuf> {
+    let config_dir: PathBuf = ["src", "tools", "dashboard", "configs"].iter().collect();
+    let mut config_paths = HashSet::new();
+    let mut stack = vec![config_dir];
+    while !stack.is_empty() {
```

Comment:
There's a walkdir crate and https://doc.rust-lang.org/1.2.0/std/fs/fn.walk_dir.html which seems to be marked unstable... are we able to use these? (Maybe the answer is no)

Reply:
It looks like the `fs::walk_dir` got deprecated in favor of the crate. Replaced the code with the crate. Thanks!